keps/sig-scheduling/20190312-pod-affinity-and-anti-affinity-support-gt-and-lt-operators.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Pod affinity/anti-affinity supports Gt and Lt operators
+authors:
+  - "@wgliang"
+owning-sig: sig-scheduling
+reviewers:
+  - "@bsalamat"
+  - "@k82cn"
+  - "@Huang-Wei"
+approvers:
+  - "@bsalamat"
+  - "@k82cn"
+creation-date: 2019-02-22
+last-updated: 2019-03-12
+status: provisional
+---
+
+ # Pod affinity/anti-affinity supports Gt and Lt operators
+
+ ## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+  * [User Stories](#user-stories)
+  * [Risks and Mitigations](#risks-and-mitigations)
+* [Design Details](#design-details)
+  * [Content](#content)
+  * [Test Plan](#test-plan)
+  * [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+ ## Summary
+
+ Extend the `Pod` affinity/anti-affinity operators to support `Gt` and `Lt` to provide 
+ users with more elegant Pod label selection capabilities.
+
+ ## Motivation
+
+  We know that `Node` affinity/anti-affinity currently supports `In`, `NotIn`, `Exists`, 
+`DoesNotExist`, `Gt`, `Lt`. But Pod affinity/anti-affinity only works with regular 
+label selectors: `In`, `NotIn`, `Exists`, `DoesNotExist`.
+
+ This is not an ideal situation if users want to put pods based on the label range. 
```

Comment:
[general comment, not api specific] Can you please give a specific example use case that this KEP would enable? I understand the appeal of making it consistent, but why will users care?

Reply:
Ah, I see there are user stories below--thanks, sorry for the noise.